signer/src/transaction_coordinator.rs

```diff
@@ -400,14 +402,28 @@ where
             tracing::debug!("no requests to handle, exiting");
             return Ok(());
         };
-
         tracing::debug!(
             num_deposits = %pending_requests.deposits.len(),
             num_withdrawals = pending_requests.withdrawals.len(),
             "fetched requests"
         );
         // Construct the transaction package and store it in the database.
         let transaction_package = pending_requests.construct_transactions()?;
+        // Get the requests from the transaction package because they have been split into
+        // multiple transactions.
+        let sbtc_requests = SbtcRequestsContextMessage {
+            requests: transaction_package
+                .iter()
+                .map(|tx| SbtcRequestsContext::from(&tx.requests))
+                .collect(),
+            fee_rate: pending_requests.signer_state.fee_rate,
+            last_fees: pending_requests.signer_state.last_fees.map(Into::into),
+        };
+
+        // Share the list of requests with the signers.
+        self.send_message(sbtc_requests, bitcoin_chain_tip).await?;
+        // Wait for the signers to receive and process the requests.
+        tokio::time::sleep(tokio::time::Duration::from_secs(1)).await;
```

Comment:
Currently, the simplest solution is to wait for a fixed time.

An possible approach could be switching to an ACKs and maintaining a count for each signhash indicating whether it is accepted or rejected. We would stop as soon as one of the following conditions is met:

All signhashes exceed the threshold of sign promises -> proceed with the signing round.
Any signhash reaches a rejection count that prevents consensus -> skip the signing round (should we invalidate the request?).
Maximum time has elapsed -> attempt the signing round anyway?

Reply:
I would simplify it a bit for our initial implementation and just wait for the threshold number of Acks (with a timeout) and proceed with a signing round after the timeout.

We should design the Ack message though to be like you said, with a vector transactions IDs, sighashs, and a boolean for whether they will sign it. And I don't think that we need the transaction IDs so maybe we don't Ack with those.

Reply:
But we don't need to do all of that in this PR so this is fine.